tests/func/test_gc.py

```diff
@@ -204,3 +205,20 @@ def test_gc_no_dir_cache(tmp_dir, dvc, repo_template):
 
 def _count_files(path):
     return sum(len(files) for _, _, files in os.walk(path))
+
+def test_gc_no_unpacked_dir(tmp_dir, dvc, repo_template):
+    os.mkdir("test1")
+    fd = open("test1/foo", 'w')
+    fd.write("foo")
+    fd.close()
```

Comment:
This is already taken care by [`repo_template`](https://github.com/iterative/dvc/blob/8041d70552f04137a232e09227be8ff76ffc6c51/tests/dir_helpers.py#L240) in following structure:
https://github.com/iterative/dvc/blob/8041d70552f04137a232e09227be8ff76ffc6c51/tests/dir_helpers.py#L58-L65

So, I think, this should just `dvc.add(["dir"])`.

Reply:
Done.